app/src/main/java/com/example/bigowlapp/utils/ScheduledLocationTrackingManager.java

```diff
@@ -21,10 +21,10 @@
 public class ScheduledLocationTrackingManager {
 
     private static final int REQUEST_CODE = 2;
-    private static final int DEFAULT_TRACKING_RADIUS_METERS = 300;
-    private static final long DEFAULT_TRACKING_EXPIRE_TIME_MILLIS = 30L * Constants.MINUTE_TO_MILLIS;
+    private static final int TRACKING_RADIUS_METERS = 300;
```

Comment:
We could turn this into ENUMS as I feel this could be use elsewhere in the future.

Reply:
I would disagree since enums are usually used to group together multiple states of one value, ex: a color would have enums RED, BLUE for example.

The constants of this class are each a value of their own respective things, and their only relation is to the class itself, hence they are static.

If you disagree, maybe explain what the enum would represent, because maybe I just didn't understand what you were saying.

Reply:
I rectified myself about the enums, I meant constants.

Reply:
Just did the change